rlbot_gui/bot_management/bot_creation.py

```diff
@@ -15,37 +17,38 @@ def convert_to_filename(text):
     """
     import unicodedata
     normalized = unicodedata.normalize('NFKD', text).encode('ascii', 'ignore').decode()
-    valid_chars = "-_.() %s%s" % (string.ascii_letters, string.digits)
+    valid_chars = '-_.() %s%s' % (string.ascii_letters, string.digits)
     filename = ''.join(c for c in normalized if c in valid_chars)
     filename = filename.replace(' ', '_')  # Replace spaces with underscores
     return filename
 
 
 def bootstrap_python_bot(bot_name, directory):
-    bot_directory = directory or "."
     sanitized_name = convert_to_filename(bot_name)
+    bot_directory = Path(directory or '.')
 
     with tempfile.TemporaryDirectory() as tmpdirname:
-        print('created temporary directory', tmpdirname)
+        tmpdir = Path(tmpdirname)
+        print('created temporary directory', tmpdir)
 
         download_and_extract_zip(
-            download_url="https://github.com/RLBot/RLBotPythonExample/archive/master.zip",
-            local_zip_path=f"{tmpdirname}/RLBotPythonExample.zip", local_folder_path=tmpdirname)
+            download_url='https://github.com/RLBot/RLBotPythonExample/archive/master.zip',
```

Comment:
Single versus double quotes should be discussed and applied as project-wide change in a new PR, TODO

Reply:
Acknowledged. Maybe introducing an auto-fixing linter would be the most effective tool to do this.
I've made things consistently use single quotes here as that was the dominant quoting style in this file. I don't have strong opinions on which quotes to use (although I prefer single quotes) but like things being consistent.

Reply:
Created #27 to continue this conversation